scripts/test/enzyme.setup.js

```diff
@@ -1,12 +0,0 @@
-/* eslint import/no-extraneous-dependencies: [error, { devDependencies: true }] */
-
-import 'regenerator-runtime/runtime';
```

Comment:
Seems like `regenerator-runtime` package is not used anywhere else, so we can remove it as well

Reply:
:+1: